docs/source/VUDRM-key-provision.md

```diff
@@ -505,3 +505,128 @@ The values are:
 - `cupertinostreaming-aes128-iv-include-in-chunklist`: Must be set to `false`.
 - `cupertinostreaming-aes128-key-format`: Must be set to `com.apple.streamingkeydelivery`.
 - `cupertinostreaming-aes128-key-format-version`: Must be set to `1`.
+
+### Vualto Wowza DRM API
+
+The Vualto Wowza DRM API is a small Docker web API application which runs on the Wowza server, allowing the provisioning of keyfiles through a simple REST interface.
+
+#### Running using Docker
+
+In this example, the API is exposed on port 9000 and the API key is set to some GUID:
+
+```
+docker run -d --restart always -p 9000:80 \
+-e WOWZADRMAPI_APIKEY=93640045-31f1-45c0-aa0d-b5682d7c9ac8 \
+-e WOWZADRMAPI_KEYFILESPATH=/mnt/keyfiles \
+-e WOWZADRMAPI_GRANDCENTRALURL=https://config.vudrm.tech/v1/clients \
+-e WOWZADRMAPI_KEYPROVIDERURL=https://keyprovider.vudrm.tech \
+-v /usr/local/WowzaStreamingEngine-4.7.7/keys:/mnt/keyfiles \
+--name wowza-drm-api vualto/wowza-drm-api:latest
```

Comment:
should this have a version number other than "latest"? It is not a good idea to use "latest" in prod!